### DISCUSSION TITLE
Light Show Sync?

### DISCUSSION THREAD

**Discussion by davearias** [aqualinkd/AqualinkD#199] [Q&A]
Greetings all!

I have been playing around with AqualinkD controlling my lights, and I think I finally got the timings where it works reliably. One question: When selecting a "Show", is AqualinkD capable of syncing multiple channels running the same show? My RS8 panel will do this, by starting it's programming cycle over with any channel running the light show. AqualinkD doesn't seem to do this?

Maybe there's a way to do it, and I'm missing it. Or change the program on multiple light channels simultaneously?

Thanks!

**Comment by sfeakes** [comment]
AqualinkD is not capable of programming multiple lights at the same time if the lights have a separate relay to control them.
IE, I have multiple lights that AqualinkD can sync, but only because they are connected to the same relay in the control box. (I think the is is what you mean by “channel”).

I actually did not know an RS8 panel could do this, I will take a look at how it does this and see if if can be replicated in AqualinkD

**Reply by davearias** [reply]
You're correct in my terminology: channel=circuit=relay=group of lights. I blame my entertainment background.

My pool has three light relays, pool, spa, bubblers. Let's say I have "Mardi Gras" light show running in the pool, and I want to then change the spa light to also run "Mardi Gras". What the panel then does is reset both groups of lights back to program 1, and then cycles the relays simultaneously so that once they get to "Mardi Gras", they change color in sync. I would assume all panels do this when configured with multiple color changing lights?

On a similar note, whenever AqualinkD is running, the timing for relay cycles is inconsistent, and what will happen is sometimes a relay is switched off long enough for the lights to reset back to color program 1. Because of this, I can never guarantee that I will get the color program I selected. This happens if AqualinkD is running and I change the light program through iAqualink App or AqualinkD (via MQTT). I have tested this with the light being set up as a Jandy WaterColor light in the panel, and with the light set up as a simple relay switch. If I stop AqualinkD, everything works as expected. This is my last hurdle to getting away from iAqualink.

I'm running on a RPi Model B rev 2, with the recommended FTDI USB/RS485 adapter, and a USB WiFi adapter, Debian Buster, AqualinkD v2.2.2. I *love* what I get with AqualinkD, but the light control is driving my wife nuts.

Thanks!

**Reply by superdupe** [reply]
I can confirm this is exactly how my controller would operate with my older Jandy WaterColor lights.  They died and I replaced them with J&J ColorSplash...  They are supposed to be exactly like the Pentair 5G, but when I set them up in the controller as Pentair, I couldn't get them to sync.  Maybe its just a Jandy WaterColor thing?  Anyway, +1 vote for anything more we can do to get away from the Jandy interface would be awesome!
